auto_round/special_model_handler.py

```diff
@@ -36,6 +36,57 @@
 }
 SPECIAL_SHARED_CACHE_KEYS["MiniMaxText01ForCausalLM"] = ("slope_rate",)
 
+CONVERT_EXPERT_TO_LINEAR_MODELS = ["llama4"]
+
+
+def _get_moe_converter(config):
+    import torch
+    from transformers.modeling_utils import no_init_weights
+
+    # https://github.com/vllm-project/llm-compressor/blob/main/src/llmcompressor/modeling/llama4.py
+    if config.model_type == "llama4":
+        from transformers.models.llama4.modeling_llama4 import Llama4TextMLP
```

Comment:
better log a warning, the quantized model could not run transformers

Reply:
add print